vignettes/MsBackend.Rmd

```diff
@@ -96,12 +96,21 @@ General conventions for MS data of a `Spectra` are:
 
 ## Notes on parallel processing
 
-The default parallel processing of `Spectra` splits backends by their
-`dataStorage` and performs the processing in parallel on the subsets. Not all
-backends will support such parallel processing (or any parallel processing) and
-thus unexpected errors might occur (depending on the user's settings). A new
-function allowing to evaluate the parallel processing capabilities of a backend
-will be added to help avoiding such issues.
+For parallel processing, `Spectra` splits the backend based on a defined
+`factor` and processes each in parallel (or *in serial* if a `SerialParam` is
+used). The splitting `factor` can be defined for `Spectra` by setting the
+parameter `processingChunkSize`. Alternatively, through the
+`backendParallelFactor` function the backend can also *suggest* a `factor` that
+should/could be used for splitting and parallel processing. The default
+implementation for `backendParallelFactor` is to return an empty `factor`
+(`factor()`) hence not suggesting any preferred
+splitting. `backendParallelFactor` for `MsBackendMzR` on the other hand returns
+a `factor` based on the data files the data is stored in (i.e. based on the
+`dataStorage` of the MS data).
+
+Besides parallel processing, this chunk-wise processing can also reduce the
```

Comment:
I know it's very much implied but here or in the chunk processing vignette maybe explicitly say that it is advised to set up for big dataset that do not have `MsBackendMzR` type of backend. Just so that less experienced user get the important info.

Reply:
yes! thanks! I will also add a dedicated vignette on that topic.